bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/DahuaHandler.java

```diff
@@ -43,12 +43,13 @@
 @NonNullByDefault
 public class DahuaHandler extends ChannelDuplexHandler {
     private IpCameraHandler ipCameraHandler;
-    private int nvrChannel;
+    private int nvrChannelAdjusted;
     private Pattern boundaryPattern;
 
     public DahuaHandler(IpCameraHandler handler, int nvrChannel) {
         ipCameraHandler = handler;
-        this.nvrChannel = nvrChannel;
+        // Most of the API is the NVR channel -1, but some of it is not, like streams and snapshot URLS.
```

Comment:
I guess this is some 0 or 1 based index issue. Just double checking, are all calls tested and confirmed to be working as expected? As this change is error prone.

Reply:
Yes they are checked and I worked directly from the API.